R/plot_with_settings.R

```diff
@@ -144,8 +144,9 @@ plot_with_settings_ui <- function(id) {
 #'       } else if (input$download_option == "trellis") {
 #'         lattice::densityplot(1)
 #'       } else if (input$download_option == "grob") {
-#'         tr_plot <- lattice::densityplot(1)
-#'         ggplotify::as.grob(tr_plot)
+#'         grid.newpage()
+#'         l <- linesGrob()
+#'         grid.draw(l)
```

Comment:
```suggestion
#'         grid::grid.newpage()
#'         l <- grid::linesGrob()
#'         grid::grid.draw(l)
```

Grid is a base package but not the startup one.